### PR TITLE
Add raw data download to FETP

### PR DESCRIPTION
**Pull request by PShoe**
### Changes:
- Add dashboard group
- Add dashboard reports to Solomon Islands and PNG

Note: I suspect there is a less repetitive way of adding same report to two countries.

---

### Screenshots:

image

### CODE REVIEW

**Comment by nick-beyondessential** [comment]
Also better practise to link screenshots to card rather than PR in case they contain sensitive data.

**Comment by PShoe** [comment]
@nick-beyondessential back to you for review, switched to rawDataValues - chat to me about this tomorrow?

File: packages/database/src/migrations/20210214234630-AddDataDownloadNationalFETP-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DATA_ELEMENT_CODE = 'FGDS';
```
Comment (nick-beyondessential) [inline packages/database/src/migrations/20210214234630-AddDataDownloadNationalFETP-modifies-data.js:19]: Bit pedantic but I'd rename this variable `SURVEY_CODE`

**Review by nick-beyondessential — APPROVED** [review]
Looks good, optional name suggestion

File: packages/database/src/migrations/20210214234630-AddDataDownloadNationalFETP-modifies-data.js (hunk 1 above)
Comment (PShoe) [reply packages/database/src/migrations/20210214234630-AddDataDownloadNationalFETP-modifies-data.js]: done, thanks!

**Pull request merged**